src/test/java/de/tum/in/www1/artemis/AttachmentUnitIntegrationTest.java

```diff
@@ -16,6 +16,7 @@
 import de.tum.in.www1.artemis.repository.*;
 import de.tum.in.www1.artemis.util.ModelFactory;
 
+@Deprecated // Moved to Lecture microservice. To be removed
```

Comment:
Remove, see lecture unit PR

Reply:
I'll remove the @ Deprecated and will only leave the comment so that it's easy to find later when it's time to remove this file